### PR TITLE
resolved issue 174

### LINKED ISSUE #174: zip file not available to download
**Describe the bug**

Zip file download for Working or Clean dataset are not working for this particular WQP dataset.

**To Reproduce**

1. Query WQP for data using the following filters:

- startDate = "2019-04-15",
- endDate = "2023-04-15",
- sampleMedia = c("Water", "water"),
- siteType = c("Stream", "Lake, Reservoir, Impoundment"),
- statecode = "WY"

This may take a few minutes to run.

2. Try to download the Working Dataset from the overview tab. 

The zip file is not available. This is strange because it is the only dataset I've run into this particular issue with. You can still run through other features/tabs in the app, but the download does not work. It tries to download the data as a HTML Document (*.htm).

**Expected behavior**

Use should be able to download a .zip file.

**Screenshots**

image

image

**Reminders for TADA contributors addressing this issue**

Bug fixes should include the following work:

-   [ ] Create or edit the code.

-   [ ] Document all code using line/inline and/or multi-line/block comments
    to describe what is does.

-   [ ] Create or edit tests in tests/testthat folder to help prevent and/or 
    troubleshoot potential future issues.

-   [ ] If your code edits impact other functionality in the shiny 
    app, ensure those are updated as well.

-   [ ] Run styler::style_pkg(), devtools::document(), and devtools::check() 
    and address any new notes or issues before creating a pull request.

### PR DESCRIPTION
**Pull request by triphook**

### CODE REVIEW

File: R/mod_TADA_summary.R [hunk 1]
```diff
@@ -205,7 +205,9 @@ mod_TADA_summary_server <- function(id, tadat) {
         setwd(tempdir())
         datafile_name <- paste0(tadat$default_outfile, ".xlsx")
         progress_file_name <- paste0(tadat$default_outfile, "_prog.RData")
+        print(111)
```
Comment (cristinamullin) [inline R/mod_TADA_summary.R]: @triphook ok to remove these prints?

File: R/utils_track_progress.R [hunk 2]
```diff
@@ -159,32 +162,33 @@ writeNarrativeDataFrame <- function(tadat) {
...
     df[nrow(df) + 1, ] <- c("Depth unit conversion", "None")
   }
 
-
   # Filtering tab
-  for (row in 1:nrow(tadat$selected_filters)) {
-    df[nrow(df) + 1, ] <- c(
-      "Selected Filter",
-      paste0(
+  if (nrow(tadat$selected_filters) > 0) {
```
Comment (cristinamullin) [inline R/utils_track_progress.R:180]: Is this edit fixing a different issue you ran into?

File: R/mod_TADA_summary.R (hunk 1 above)
Comment (triphook) [reply R/mod_TADA_summary.R]: Yes

File: R/utils_track_progress.R (hunk 2 above)
Comment (triphook) [reply R/utils_track_progress.R:180]: This was one of the first things I tried to resolve the zipfile issue. It didn't fix it, but I think it's a good idea to keep it. This just skips the recording of selected filters in the progress file if the filters haven't been selected yet.

**Review by cristinamullin — APPROVED** [review]

**Pull request merged**